README.md

```diff
@@ -10,6 +10,7 @@ Welcome to the Intro to Open Source Course with OpenSauced! This course is desig
 ## Language Support
```

Comment:
Do we really need this section here?

Reply:
I think it could help @antonio-pedro99. Languages in the context of tech and open source tends to be used to describe React, TypeScript, etc, so describing it as a translation adds clarity. Maybe, we can just reword it to check the navbar on top? What do you think @geoffreylgv? @BekahHW?

Reply:
Hi @BekahHW  @CBID2 @antonio-pedro99 , 

talking about the `## Language support` in the README, yes we do need it.
Even if we have docsify showing the files as HTML in the browser, and we have the language dropdown for language, it doesn't exclude to have this in the README of the repo.

It really shows that we have multi-language to the course on GitHub and everyone seeing it, will be glad & will contribute with another language who does not appear.

What we can do to improve and do not have a thousand of bullet point, is to align language flags only instead of writing the language. 

All the flag can be found here : "https://cdn.statically.io/flags/"+"lang_code"+".svg"
`lang_code` is the country language code who is [Alpha 2 code](https://www.iban.com/country-codes), do not forget the extension `.svg`

Example:
|Country| lang_code (Alpha 2 code)|Flag URL|
|------|-----------|--------|
|Togo|tg|`https://cdn.statically.io/flags/tg.svg`|
|United States of America|us|`https://cdn.statically.io/flags/us.svg`|
|France|fr|`https://cdn.statically.io/flags/fr.svg`|
|Brazil|br|`https://cdn.statically.io/flags/br.svg`|

This example will clarify what I was saying earlier:
---

<div align="center">
  <br>
  <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
  <h1>🍕 Intro to Open Source with OpenSauced 🍕</h1>
  <strong>The site provides course materials for OpenSauced's Intro to Open Source.</strong>
</div>
<br>

Welcome to the Intro to Open Source Course with OpenSauced! This course is designed to provide you with an introduction to open source and guide you through the process of contributing to open source projects.
## Language Support
## Language Support
Like pizza, open source is meant to be shared with everyone. Click on the links below to take this course in the following languages:
<kbd>[<img title="French" alt="French" src="https://cdn.statically.io/flags/fr.svg" width="22">](translations/fr/README.md)</kbd><kbd>[<img title="English" alt="English" src="https://cdn.statically.io/flags/us.svg" width="22">](translations/fr/README.md)</kbd><kbd>[<img title="Portugues Brasil" alt="Portugues Brasil" src="https://cdn.statically.io/flags/br.svg" width="22">](translations/fr/README.md)</kbd>


---
```bash

<div align="center">
  <br>
  <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
  <h1>🍕 Intro to Open Source with OpenSauced 🍕</h1>
  <strong>The site provides course materials for OpenSauced's Intro to Open Source.</strong>
</div>
<br>

Welcome to the Intro to Open Source Course with OpenSauced! This course is designed to provide you with an introduction to open source and guide you through the process of contributing to open source projects.
## Language Support
## Language Support
Like pizza, open source is meant to be shared with everyone. Click on the links below to take this course in the following languages:
<kbd>[<img title="French" alt="French" src="https://cdn.statically.io/flags/fr.svg" width="22">](translations/fr/README.md)</kbd>
<kbd>[<img title="English" alt="English" src="https://cdn.statically.io/flags/us.svg" width="22">](translations/fr/README.md)</kbd>
<kbd>[<img title="Portugues Brasil" alt="Portugues Brasil" src="https://cdn.statically.io/flags/br.svg" width="22">](translations/fr/README.md)</kbd>
```

Reply:
@geoffreylgv thanks for the insights

Reply:
> @geoffreylgv thanks for the insights

You're welcome @antonio-pedro99

Reply:
@antonio-pedro99, you'll do the Portuguese one. I'll add this change to the English file. @geoffreylgv, you do the French one.  Sound good?

Reply:
> @antonio-pedro99, you'll do the Portuguese one. I'll add this change to the English file. @geoffreylgv, you do the French one. Sound good?

Yeah, sure